### DISCUSSION TITLE
Projects (Beta): Promote Labels to Fields?

### DISCUSSION THREAD

**Discussion by DHowett** [community/community#7225] [Projects and Issues]
My team (and likely many others) makes heavy use of labels, for all sorts of things:

* Priority (P0-P3)
* Product (Which of the multiple toplevel needs served by the repository an issue is part of)
* Area (Where in the Product the issue is)
* Type (Bug, Task, ...)
* Current resolution (Duplicate, External, Fixed, ...)

An issue on our repository might look like this:

<img width="477" alt="image" src="https://user-images.githubusercontent.com/189190/140174207-2150fa06-5f41-4c3c-ad70-f75d08e31f31.png">

Given that we have a thousand or more issues tagged up with all of these labels -- sometimes multiple labels in each category (for cross-cutting issues that span multiple areas or sub-areas, or multiple products from the same repository) -- it is difficult for us to move to the new Projects (beta) table layout.

Right now, all of our issues look somewhat more like this:

<img width="1128" alt="image" src="https://user-images.githubusercontent.com/189190/140174632-92c01baa-0969-437a-ad14-451e7e0571f4.png">

Each piece of data is already represented in the Labels column, but that column cannot be used for further insights, filtering or sorting.

It would be great if we could set label->field mappings and automatically promote all of our labels into fields.

**Comment by Jason-Varghese-form3** [comment]
Is there a Fix for this? I have the same suggestion/request!
